http://dev.mutt.org/trac/changeset/fe30f394cbe6

Not being able to start an IMAP command is a fatal error.
Closes #3155.

Index: imap/command.c
===================================================================
--- imap/command.c (revision 5651:aac0c04727c0)
+++ imap/command.c (revision 5955:fe30f394cbe6)
@@ -234,7 +234,4 @@
 
   if ((rc = cmd_start (idata, cmdstr, flags)) < 0)
-    return rc;
-
-  if (rc < 0)
   {
     cmd_handle_fatal (idata);
